index.html

```diff
@@ -195,35 +254,42 @@ <h2>CONTACT US</h2>
                 <br />
                 <input type="email" id="email" name="email" placeholder="jeremy@gmail.com" required /><br />
```

Comment:
- [ ] [OPTIONAL] In the desktop version the contact form is not centered on the page kindly make sure it does, so your design looks more semantic and follows the Figma style

![Screenshot_20220708_044300](https://user-images.githubusercontent.com/25227022/177912765-bf7b905f-143c-4063-8c5c-6a52eadf40ab.png)